esmvalcore/experimental/recipe.py

```diff
@@ -133,7 +133,6 @@ def run(
             session["diagnostics"] = task
 
         with log_to_dir(session.run_dir):
-            _dask.check_distributed_config()
```

Comment:
Would this need to be replaced by
```python
_dask.validate_dask_config(session["dask"])
```
so users of the notebook interface also benefit from having their Dask configuration checked before running the recipe? The notebook interface does not use `esmvalcore/_main.py` to run recipes. Or could we move the check of the Dask configuration to `esmvalcore._recipe.recipe.Recipe.__init__` so the configuration is checked before going through the lengthy process of finding all the input data?

Reply:
Done in https://github.com/ESMValGroup/ESMValCore/pull/2616/commits/ab02a3587e9814e81298a3b1b47d1e2c2c5e4f8f